notebooks/outetts-text-to-speech/outetts-text-to-speech.ipynb

```diff
@@ -22,6 +22,11 @@
     "- [Run model inference](#Run-model-inference)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    demo = make_demo(interface)
</pre><p>I think it would be nice to use the quantized model in the interactive demo. Could you please add an option for the user to choose between the original and optimized models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Done.